tools/testinfra/runner/src/main.rs

```diff
@@ -32,23 +35,35 @@ struct Options {
     #[structopt(long = "buck-test-info")]
     spec: PathBuf,
 
-    /// Lists all unit tests and exits without running them
-    #[structopt(long)]
+    /// Lists unit tests and exits without running them
+    #[structopt(long, short)]
     list: bool,
 
     /// Path to generated test report in JUnit XML format
     #[structopt(long = "xml")]
     report: Option<PathBuf>,
 
+    /// Connection string of the DB used in stateful test runs
```

Comment:
This isn't entirely true though? Ideally we could have a default readonly connection so that non-recording runs (eg local users) can also get the list of disabled tests. Or are you planning to solve that separately?

If the disabled test feature only work on CI, that is fine for now, but it would be nice for non-stateful runs to still get the readonly info from the db